meilisearch/src/routes/mod.rs

```diff
@@ -327,3 +327,5 @@ pub async fn get_health(
 
     Ok(HttpResponse::Ok().json(serde_json::json!({ "status": "available" })))
 }
+
+mod multi_search;
```

Comment:
Can you move this `mod` to the top of the file with the other ones

Reply:
done